app/assets/stylesheets/expressive.scss

```diff
@@ -0,0 +1,20 @@
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.full-width {
+  width: 100wv;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.margin-20 {
```

Comment:
i did this, because @SKOLZ  suggest  work  with [Expressive CSS](http://johnpolacek.github.io/expressive-css/), who create a class for common properties and make the code more readable. But it just a way to do it.